pyproject.toml

```diff
@@ -29,6 +29,7 @@ dependencies = [
   "psutil",
   "click",
   "pyyaml~=6.0",
+  "kubernetes",
```

Comment:
I think we need a >=29 on this to get the CI tests to pass.

P.S. We can probably remove a bunch of the test dependencies related to formatting and all that right now.

Reply:
done